app/models.py

```diff
@@ -162,9 +162,23 @@ class AliasGeneratorEnum(EnumE):
     uuid = 2  # aliases are generated based on uuid
 
 
+class HibpDataClass(db.Model, ModelMixin):
+    __tablename__ = "hibpdataclass"
+    description = db.Column(db.Text, nullable=False, unique=True, index=True)
```

Comment:
I wonder if we can use "name" or "attribute" instead of "description" here 🤔. "description" usually means a long text.

Reply:
Heh, funny. I avoided name because you explicitly didn't like me using name for the Hibp model, but you accepted that because "that's how HIBP calls it". This HIBP doesn't give any explicit name, but I can change it. Just tell me if you want name or attribute.

Reply:
Oh sorry, I don't like "name" neither as it doesn't convey uniqueness ... Let's try with "attribute" then as it seems that HIBP uses this word.